components/speedreader/speedreader_component.cc

```diff
@@ -8,9 +8,7 @@
 #include <utility>
 
 #include "base/bind.h"
-#include "base/command_line.h"
 #include "base/logging.h"
-#include "brave/components/speedreader/speedreader_switches.h"
 #include "url/gurl.h"
```

Comment:
seems not used

Reply:
Removed in https://github.com/brave/brave-core/pull/10090/commits/bab3ce896441e392eb05e54d46c2133dcd5b5a60